Makefile

```diff
@@ -1153,18 +1151,12 @@ kind/port-forwards:
 		&& cd lagoon-charts.kind.lagoon \
 		&& $(MAKE) port-forwards
 
-# Use kind/retest to only perform a push of the local-dev and test images, and
-# run the tests. It preserves the last build lagoon core & remote setup, reducing
-# rebuild time.
+# kind/retest re-runs tests in the local cluster. It preserves the last build
+# lagoon core & remote setup, reducing rebuild time.
 .PHONY: kind/retest
 kind/retest:
-		export KUBECONFIG="$$(pwd)/kubeconfig.kind.$(CI_BUILD_TAG)" && \
-		export IMAGE_REGISTRY="registry.$$(./local-dev/kubectl get nodes -o jsonpath='{.items[0].status.addresses[0].address}').nip.io:32080/library" \
-		&& docker login -u admin -p Harbor12345 $$IMAGE_REGISTRY \
-		&& for image in $(KIND_TESTS); do \
-			docker tag $(CI_BUILD_TAG)/$$image $$IMAGE_REGISTRY/$$image:$(SAFE_BRANCH_NAME) \
-			&& docker push $$IMAGE_REGISTRY/$$image:$(SAFE_BRANCH_NAME); \
-		done \
+	export KUBECONFIG="$$(pwd)/kubeconfig.kind.$(CI_BUILD_TAG)" \
+		&& export IMAGE_REGISTRY="registry.$$(./local-dev/kubectl get nodes -o jsonpath='{.items[0].status.addresses[0].address}').nip.io:32080/library" \
```

Comment:
we need to keep a push-images routine in here to push the TEST images - that's mostly when I've been using kind/retest - to iterate ansible playbooks etc.

Reply:
So yes we can do this but what I was going for was a workflow like this for iterating `tests`:

```
rm build/tests
make -j8 build/tests SKIP_SCAN=true
make kind/push-images IMAGES=tests
make kind/retest TESTS='[features-api-variables]'
```

The advantage is that if you are iteratively working on something other than the `tests` image (e.g. `kubectl-build-deploy-dind`) and don't need to push the `tests` image all the time, then you can selectively rebuild and push only the image that you are actually interested in.

Does that make sense?

Reply:
yes, that does (I just read the docs update too...  Happy for this!